src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --blue: #00acdc;
+}
+
+.moyoH {
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.list-style {
+  display: flex;
+  justify-content: space-between;
+}
+
+.mLogo {
+  height: 40px;
+  width: 40px;
+  padding-top: 10px;
+}
+
+.moyo_list {
+  display: flex;
+  list-style: none;
+}
+
+.moyo_item {
+  margin-left: 20px;
+}
+
+.moyo_item:first-child {
+  margin-left: 0;
+}
+
+.moyo_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
```

Comment:
move to html selector

Reply:
Means, move `font-family` to `html` selector